### PR TITLE
New Presence: AnimeTube

### PR DESCRIPTION
**Pull request by Drakonir**
Features:
- Show Browsing Status
- Show Anime Info
- Show Time remaining on Episode
- Show User Profiles

Future Features:
- Buttons
- More Details

Working Proof:
AnimeTubeTimeWorking
ProfileAnimeTubeWorking

### CODE REVIEW

File: websites/A/AnimeTube/presence.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+let videoData: {
+  duration: number,
+  paused: boolean,
+  currentTime: number
+};
+
+presence.on("iFrameData", (data: typeof videoData) => {
+    if (data !== null)
+    videoData = data;
```
Comment (ririxidev) [inline websites/A/AnimeTube/presence.ts:18]: what? how this if works then?

File: websites/A/AnimeTube/presence.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
+const presence = new Presence({
+    clientId: "833689728774832168"
+  }),
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+
```
Comment (ririxidev) [inline websites/A/AnimeTube/presence.ts:7]: dont need this blank line

File: websites/A/AnimeTube/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "AnimeTube",
+    "author": {
+      "name": "Drakonir",
+      "id": "469868455902969866"
+    },
+    "category": "anime",
+    "logo": "https://i.imgur.com/9MxXOK3.png ",
```
Comment (ririxidev) [inline websites/A/AnimeTube/dist/metadata.json:10]:
```suggestion
    "logo": "https://i.imgur.com/9MxXOK3.png",

```

**Review by ririxidev — CHANGES_REQUESTED** [review]
also why ``|`` in details, and what ``state`` is meaning tbh, bc i see sometimes 
image
etc. I'm kinda confused and I think it isn't necessary to have in presence.

**Comment by Drakonir** [comment]
> 
> 
> also why `|` in details, and what `state` is meaning tbh, bc i see sometimes
> image
> etc. I'm kinda confused and I think it isn't necessary to have in presence.

It was added so it seemed more fun, ill edit them to something more informative
the | is just for looks!

File: websites/A/AnimeTube/dist/metadata.json [hunk 4]
```diff
@@ -7,7 +7,8 @@
       "id": "469868455902969866"
     },
     "category": "anime",
-    "logo": "https://i.imgur.com/9MxXOK3.png ",
+    "logo": "https://i.imgur.com/9MxXOK3.png",
+
```
Comment (ririxidev) [inline websites/A/AnimeTube/dist/metadata.json:11]: it added new line 😂

File: websites/A/AnimeTube/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{
+
+    "$schema": "https://schemas.premid.app/metadata/1.3",
```
Comment (Bas950) [inline websites/A/AnimeTube/dist/metadata.json:3]:
```suggestion
{
    "$schema": "https://schemas.premid.app/metadata/1.3",
```

File: websites/A/AnimeTube/presence.ts (hunk 1 above)
Comment (Drakonir) [reply websites/A/AnimeTube/presence.ts]: Removed as it was not needed

**Review by Drakonir — COMMENTED** [review]
Messed up the Title

The if was removed as it was not needed

Everything should be good now

**Review by Bas950 — APPROVED** [review]

**Pull request merged**